package.json

```diff
@@ -90,6 +91,7 @@
     "ember-computed-decorators": "^0.2.2",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-cli-sass": "5.6.0",
+    "ember-frost-core": "^1.0.0",
```

Comment:
Yeah, that's fine - I doubt we have any scenarios where only the nav is used currently anyway